packages/@uppy/core/src/index.test.js

```diff
@@ -1020,7 +1020,7 @@ describe('src/Core', () => {
       })
 
       core._calculateTotalProgress()
-      expect(core.getState().totalProgress).toEqual(65)
+      expect(core.getState().totalProgress).toEqual(66)
     })
```

Comment:
65 vs 66? 🤔

Reply:
Progress was `Math.floor`ed in the past but I used `Math.round` here

Reply:
Thanks for the explanation! :)